apps/portal/extensions/components/gadget/index.js

```diff
@@ -121,6 +128,7 @@
     };
 
     component.update = function (sandbox, comp, hub, done) {
+        console.log("8----------------------------------------");
```

Comment:
@ayshsandu 
Can you provide a proper log for this if a log is needed without '8-----' like log? Also is it correct to use console.log in js? I don't think it is supported by all browsers